Python 3 functions and lambdas cannot take explicit tuple param

I will provide a PR for these...

```
./artemis/ml/datasets/art_gallery.py:48:18: E999 SyntaxError: invalid syntax
def get_new_size((cy, cx), new_size):
                 ^

./artemis/ml/datasets/datasets.py:109:59: E999 SyntaxError: invalid syntax
        return self.process_with(targets_processor=lambda (t, ): (encoder(t), ))
                                                          ^

./artemis/ml/predictors/train_and_test.py:169:27: E999 SyntaxError: invalid syntax
    def __getitem__(self, (data_subset, prediction_function_name, cost_name)):
                          ^

./artemis/plotting/matplotlib_backend.py:260:28: E999 SyntaxError: invalid syntax
def _update_axes_bound(ax, (left, right), (lower, upper), mode = 'fit'):
                           ^
```

--- a/artemis/ml/datasets/art_gallery.py
+++ b/artemis/ml/datasets/art_gallery.py
@@ -45,9 +45,10 @@ def get_image(name, size = None):
     return im_array
 
 
-def get_new_size((cy, cx), new_size):
-    ny, nx = (None, None) if new_size is None else new_size
-    aspect_ratio = float(cy)/cx
+def get_new_size(cy_and_cx, new_size):
+    cy, cx = cy_and_cx
+    ny, nx = new_size or (None, None)
+    aspect_ratio = float(cy) / cx
     if ny is None and nx is None:
         return cy
     elif nx is None:

--- a/artemis/ml/datasets/datasets.py
+++ b/artemis/ml/datasets/datasets.py
@@ -106,7 +106,7 @@ class DataSet(object):
         if n_categories is None:
             n_categories = self.n_categories  # Will throw an exception if not a categorical target
         encoder = OneHotEncoding(n_categories, form=form, dtype=dtype)
-        return self.process_with(targets_processor=lambda (t, ): (encoder(t), ))
+        return self.process_with(targets_processor=lambda t_tuple: (encoder(t_tuple[0]), ))
 
 
 class DataCollection(object):

--- a/artemis/ml/predictors/train_and_test.py
+++ b/artemis/ml/predictors/train_and_test.py
@@ -166,10 +166,12 @@ class ModelTestScore(object):
     def __init__(self, ):
         self.scores = OrderedDict()
 
-    def __getitem__(self, (data_subset, prediction_function_name, cost_name)):
+    def __getitem__(self, data_predfunc_cost):
+        data_subset, prediction_function_name, cost_name = data_predfunc_cost
         return self.scores[data_subset, prediction_function_name, cost_name]
 
-    def __setitem__(self, (data_subset, prediction_function_name, cost_name), value):
+    def __setitem__(self, data_predfunc_cost, value):
+        data_subset, prediction_function_name, cost_name = data_predfunc_cost
         self.scores[data_subset, prediction_function_name, cost_name] = value
 
     def keys(self):

--- a/artemis/plotting/matplotlib_backend.py
+++ b/artemis/plotting/matplotlib_backend.py
@@ -3,8 +3,8 @@ from itertools import cycle
 
 from artemis.config import get_artemis_config_value
 from artemis.general.should_be_builtins import bad_value
-from artemis.plotting.data_conversion import put_data_in_grid, RecordBuffer, data_to_image, put_list_of_images_in_array, \
-    UnlimitedRecordBuffer
+from artemis.plotting.data_conversion import (put_data_in_grid, RecordBuffer, data_to_image, put_list_of_images_in_array,
+    UnlimitedRecordBuffer)
 from matplotlib import pyplot as plt
 import numpy as np
 
@@ -259,7 +259,9 @@ class LinePlot(HistoryFreePlot):
         # plt.legend(loc='best', framealpha=0.5, prop={'size': self.legend_entry_size})
 
 
-def _update_axes_bound(ax, (left, right), (lower, upper), mode = 'fit'):
+def _update_axes_bound(ax, left_and_right, lower_and_upper, mode = 'fit'):
+    left, right = left_and_right
+    lower, upper = lower_and_upper
     if mode=='fit':
         ax.set_xbound(left, right)
         ax.set_ybound(lower, upper)
@@ -321,8 +323,8 @@ class Moving2DPointPlot(LinePlot):
         self._y_buffer = UnlimitedRecordBuffer() if buffer_len is None else RecordBuffer(buffer_len)
         self._x_buffer = UnlimitedRecordBuffer() if buffer_len is None else RecordBuffer(buffer_len)
 
-    def update(self, (x_data, y_data)):
-
+    def update(self, x_data_and_y_data):
+        x_data, y_data = x_data_and_y_data
         x_buffer_data = self._x_buffer(x_data)
         y_buffer_data = self._y_buffer(y_data)
 

